cfgov/unprocessed/css/molecules/pagination.less

```diff
@@ -92,6 +92,7 @@
 
     label,
     &_label {
+        line-height: @pagination-height__px;
```

Comment:
Did you test this in Firefox? I always run into trouble when trying to use line-height like this